main.sh

```diff
@@ -96,7 +91,10 @@ prepare_test_log_dir(){
 #Start DAOS agent
 start_agent(){
     echo -e "\nCMD: Starting agent...\n"
-    cmd="srun -n $SLURM_JOB_NUM_NODES $DAOS_DIR/install/bin/daos_agent -i -o $DAOS_AGENT_YAML -s /tmp/daos_agent"
+    cmd="clush --hostfile Log/$SLURM_JOB_ID/daos_all_hostlist \"
+    export PATH=$PATH; export LD_LIBRARY_PATH=$LD_LIBRARY_PATH; export CPATH=$CPATH;
+    export DAOS_DISABLE_REQ_FWD=1;
+    daos_agent -i -o $DAOS_AGENT_YAML -s /tmp/daos_agent\" "
```

Comment:
Do you have allow_insecure: true in the yaml file?  If you do then  I don't think you need the -i on the cmdline.  If you ever enable certificates then you won't need to change this file; just the yaml file.